Logstash/logstash-input-azuretopicthreadable/lib/logstash/inputs/azuretopicthreadable.rb

```diff
@@ -58,11 +70,11 @@ def process(output_queue, pid)
         if message and message.delivery_count > @deliverycount
           azure_service_bus.delete_subscription_message(message)
         end
+        Stud.stoppable_sleep(@thread_sleep_time) { stop? }
```

Comment:
Also added this based on experience of running this plugin for a few days - if it hit an exception, respect the sleep time as well, because Azure maybe throttling us.